lua/litee/symboltree/buffer.lua

```diff
@@ -44,26 +44,21 @@ function M._setup_buffer(name, buf, tab)
         vim.cmd("au WinEnter <buffer=" .. buf .. "> lua require('litee.lib.util.buffer').hide_cursor(true)")
     end
 
-    if config.scrolloff then
-        vim.cmd("au WinLeave <buffer=" .. buf .. "> lua require('litee.lib.util.buffer').set_scrolloff(false)")
-        vim.cmd("au WinEnter <buffer=" .. buf .. "> lua require('litee.lib.util.buffer').set_scrolloff(true)")
-    end
-
     -- set buffer local keymaps
     local opts = {silent=true}
-    vim.api.nvim_buf_set_keymap(buf, "n", "zo", ":LTExpandSymboltree<CR>", opts)
-    vim.api.nvim_buf_set_keymap(buf, "n", "zc", ":LTCollapseSymboltree<CR>", opts)
-    vim.api.nvim_buf_set_keymap(buf, "n", "zM", ":LTCollapseAllSymboltree<CR>", opts)
-    vim.api.nvim_buf_set_keymap(buf, "n", "<CR>", ":LTJumpSymboltree<CR>", opts)
-    vim.api.nvim_buf_set_keymap(buf, "n", "s", ":LTJumpSymboltreeSplit<CR>", opts)
-    vim.api.nvim_buf_set_keymap(buf, "n", "v", ":LTJumpSymboltreeVSplit<CR>", opts)
-    vim.api.nvim_buf_set_keymap(buf, "n", "t", ":LTJumpSymboltreeTab<CR>", opts)
-    vim.api.nvim_buf_set_keymap(buf, "n", "i", ":LTHoverSymboltree<CR>", opts)
-    vim.api.nvim_buf_set_keymap(buf, "n", "d", ":LTDetailsSymboltree<CR>", opts)
-    vim.api.nvim_buf_set_keymap(buf, "n", "H", ":LTHideSymboltree<CR>", opts)
-    vim.api.nvim_buf_set_keymap(buf, "n", "X", ":LTCloseSymboltree<CR>", opts)
-    vim.api.nvim_buf_set_keymap(buf, "n", "<Esc>", ":LTClosePanelPopOut<CR>", opts)
-    vim.api.nvim_buf_set_keymap(buf, "n", "?", ":lua require('litee.symboltree').help(true)<CR>", opts)
+    vim.api.nvim_buf_set_keymap(buf, "n", config.keymaps.expand, ":LTExpandSymboltree<CR>", opts)
```

Comment:
How do you feel about wrapping this entire block in 

```
if config.keymaps ~= nil then
 ...
end
```

That way, if user want no mapping in the buffer at all, they can provide "keymaps = nil" in their user config. I think config merging code in .setup() will merge the "keymaps = nil" key/value into the config correctly.